Dockerfile

```diff
@@ -49,13 +54,14 @@ RUN mkdir ${PROJECT_DIR} \
 
 WORKDIR /ta-lib
 
-RUN pip install numpy==1.9.2 \
+RUN pip install 'numpy>=1.11.1,<2.0.0' \
```

Comment:
Latest zipline is also on pandas 0.18.1 and scipy 0.17.1 if you want to bump those as well.